helix.el

```diff
@@ -27,6 +27,8 @@
 
 ;;; Code:
 
+(require 'simple)
```

Comment:
I believe simple.el is automatically loaded on startup so I don't think you need this require statement